packages/app/components/Forms/ChoiceButton/style.tsx

```diff
@@ -0,0 +1,62 @@
+import styled, { css } from "styled-components/native";
+import type { ChoiceButtonProps } from "./type";
+import { Theme } from "../../../provider/theme/theme.config";
+
+type ButtonType = Pick<
+  ChoiceButtonProps,
+  "error" | "isSelected" | "isSmall" | "isVertical" | "width" | "noMarginRight"
+> & { isChoice?: boolean; theme: Theme };
+
+export const Button = styled.View<ButtonType>`
+  border: ${(props) =>
+    props.isChoice
+      ? `${props.theme.forms.borderWidth} solid ${props.theme.colors.positive}`
+      : props.error
+      ? `${props.theme.forms.borderWidth} solid ${props.theme.colors.error}`
+      : `${props.theme.forms.borderWidth} solid rgba(28, 27, 37, 0.3)`};
+  background-color: ${(props) =>
+    props.isSelected ? `rgba(56, 176, 0, 0.1)` : `transparent`};
+  border-radius: 4px;
+  padding: ${(props) => (props.isSmall ? `10px 14px` : `25px`)};
+  width: fit-content;
+  flex-direction: ${(props) => (props.isSmall ? `row` : `column`)};
+  align-items: center;
+  justify-content: ${(props) => (props.isVertical ? `flex-left` : `center`)};
+  text-align: center;
+  margin-right: ${(props) =>
+    props.isSmall
+      ? props.isVertical || props.noMarginRight
+        ? `0`
+        : `10px`
+      : `0`};
+  margin-bottom: ${(props) => (props.isSmall ? `10px` : `0`)};
+  height: ${(props) => (props.isSmall ? "auto" : `100%`)};
+  width: ${(props) => props.width}px;
+`;
```

Comment:
use **styleFor** for button dimensions, margins, paddings.

Do like this, because, we need use scale func. Also we have "auto" for height, on react native, we do not have such style property.


![image](https://user-images.githubusercontent.com/102537764/173542894-dd862d55-4d1a-4f62-868b-36943e18c842.png)